finesse/config.py

```diff
@@ -53,6 +53,25 @@
 EM27_SENSORS_TOPIC = "em27_sensors"
 """The topic name to use for EM27 sensor-related messages."""
 
+DECADES_HOST = "localhost"
+"""The IP address or hostname of the DECADES server."""
+
+DECADES_SENSORS_URL = "http://{host}/decades/livedata"
```

Comment:
I think we should avoid the term sensors in reference to DECADES. Could swap SENSORS -> SERVER, or just drop SENSORS from each of these names and it would make sense.

Reply:
Done.